TravisCI JDK does not have the Oracle JCE unlimited strength policy files installed

We need to figure out a way to get the policy files installed into the TravisCI builds.

Original reported issue:
@uxcn reported and @dekobon verified that after upgrading the JDK on Linux to 1.8.0_131, CSE related unit tests are now failing:
```
(com.joyent.manta.client.crypto.MantaEncryptedObjectInputStreamTest)  Time elapsed: 0.002 sec  <<< FAILURE!
com.joyent.manta.exception.MantaClientEncryptionException:
There was a problem loading private key
Exception Context:
        [1:key_details=key=AES, algorithm=RAW]
---------------------------------
        at com.joyent.manta.client.crypto.EncryptionContext.initializeCipher(EncryptionContext.java:100)
        at com.joyent.manta.client.crypto.EncryptionContext.<init>(EncryptionContext.java:67)
        at com.joyent.manta.client.crypto.EncryptingEntity.<init>(EncryptingEntity.java:96)
        at com.joyent.manta.client.crypto.MantaEncryptedObjectInputStreamTest.encryptedFile(MantaEncryptedObjectInputStreamTest.java:975)
        at com.joyent.manta.client.crypto.MantaEncryptedObjectInputStreamTest.encryptedFile(MantaEncryptedObjectInputStreamTest.java:963)
        at com.joyent.manta.client.crypto.MantaEncryptedObjectInputStreamTest.willThrowExceptionWhenCiphertextIsAltered(MantaEncryptedObjectInputStreamTest.java:927)
        at com.joyent.manta.client.crypto.MantaEncryptedObjectInputStreamTest.willErrorIfCiphertextIsModifiedAllReadModes(MantaEncryptedObjectInputStreamTest.java:715)
        at com.joyent.manta.client.crypto.MantaEncryptedObjectInputStreamTest.willErrorIfCiphertextIsModifiedAesCtr256(MantaEncryptedObjectInputStreamTest.java:171)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
        at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
        at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
        at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
        at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
        at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
        at org.testng.TestRunner.privateRun(TestRunner.java:782)
        at org.testng.TestRunner.run(TestRunner.java:632)
        at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
        at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
        at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
        at org.testng.SuiteRunner.run(SuiteRunner.java:268)
        at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
        at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
        at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
        at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
        at org.testng.TestNG.run(TestNG.java:1064)
        at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
        at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
        at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
Caused by: java.security.InvalidKeyException: Illegal key size
        at javax.crypto.Cipher.checkCryptoPerm(Cipher.java:1039)
        at javax.crypto.Cipher.init(Cipher.java:1393)
        at javax.crypto.Cipher.init(Cipher.java:1327)
        at com.joyent.manta.client.crypto.EncryptionContext.initializeCipher(EncryptionContext.java:98)
        ... 34 more
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,6 +6,11 @@ cache:
   - $HOME/.m2
 sudo: false
 
+addons:
+  apt:
+    packages:
+      - oracle-java8-installer
+
 before_install:
   - echo 'MAVEN_OPTS="-Dorg.slf4j.simpleLogger.defaultLogLevel=warn"' >~/.mavenrc
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,6 +10,7 @@ addons:
   apt:
     packages:
       - oracle-java8-installer
+      - oracle-java8-unlimited-jce-policy
 
 before_install:
   - echo 'MAVEN_OPTS="-Dorg.slf4j.simpleLogger.defaultLogLevel=warn"' >~/.mavenrc

